values.yaml

```diff
@@ -116,6 +116,7 @@ georchestra:
     gateway:
       enabled: false
       docker_image: georchestra/gateway:latest
+      java_tool_options: "-Dspring.config.location=${georchestra.datadir}/gateway/application.yaml -Dspring.profiles.active=docker"
```

Comment:
missing the `-Dgeorchestra.datadir=...` which disappeared from the deployment spec

Reply:
good catch